package.json

```diff
@@ -129,6 +130,24 @@
           "typescript",
           "webpack"
         ]
+      },
+      {
+        "packages": [
+          "@fluentui/accessibility",
+          "@fluentui/docs",
+          "@fluentui/react-bindings",
+          "@fluentui/react-northstar",
+          "@fluentui/perf-test",
+          "@fluentui/e2e"
+        ],
+        "dependencies": [
+          "@uifabric/utilities",
+          "@fluentui/keyboard-key",
```

Comment:
I'm a bit nervous about this, especially the packages that have the same names.
1. Possibility of collisions such that other packages accidentally pull the old version not the local version. We may need some more `nohoist` settings to deal with this, though unfortunately that feature is super undocumented and may take some trial and error.
2. Even the packages with old names, it makes it easier for PRs using the old name to sneak in (hopefully it would be a lint error though)

Reply:
>1. Possibility of collisions such that other packages accidentally pull the old version not the local version. We may need some more nohoist settings to deal with this, though unfortunately that feature is super undocumented and may take some trial and error.

This will not happen because non-northstar packages will be using latest versions of internal packages as package dependencies. for northstar packages, they will be using the packages from local `node_modules`. I also double checked locally. 

>2. Even the packages with old names, it makes it easier for PRs using the old name to sneak in (hopefully it would be a lint error though)

I just double checked locally using `@uifabric/merge-styles` / `@fluentui/merge-styles`.

VSCode is suggesting the correct imports:
![image](https://user-images.githubusercontent.com/1207059/96768909-3da0ca00-1393-11eb-97e1-052733485f9b.png)

Lint rule will also catch:
![image](https://user-images.githubusercontent.com/1207059/96768972-54dfb780-1393-11eb-9a1f-1104ed447670.png)

**please help double check the dev experience though, in case I missed anything**